src/Poser.php

```diff
@@ -34,7 +34,7 @@ public function __construct($renders)
      * @param $style
      * @param $format
      */
-    public function generate(string $subject, string $status, string $color, string $style, string $format): Image
+    public function generate(string $subject, string $status, string $color, string $style, string $format = 'svg'): Image
```

Comment:
Maybe use `null` value here and assign it on `svg` if null inside of the methods? And it will be better to use `Badge::DEFAULT_FORMAT` here.

Reply:
what's the point of passing `null` and assigning value later?

Reply:
If anybody extends Poser class, then default value may be different then.

Reply:
Can't be different even now?

Reply:
I was wrong, you may overwrite default value in extended class. Just saw this way of defining default values very frequently and thought it's protecting vs this case.

Reply:
in my opinion, it's okay to set the default here, but I'd use the constant `Badge::DEFAULT_FORMAT`